### ISSUE TITLE
Bug: relative-font-units does not handle inherit

### ISSUE THREAD

**Issue by sdavids** [eslint/css#193]
### Environment

ESLint version: 9.28.0
@eslint/css version: 0.9.0
Node version: 22.16.0
npm version: 11.4.2
Operating System: Darwin redacted 21.6.0 Darwin Kernel Version 21.6.0: Mon Jun 24 00:56:10 PDT 2024; root:xnu-8020.240.18.709.2~1/RELEASE_X86_64 x86_6

### Which language are you using?

rule

### What did you do?

<details>
<summary>Configuration</summary>

```
      "css/relative-font-units": [
        "error",
        {
          allowUnits: ["rem"],
        },
      ],
```
</details>

```js
  h1 {
    font-size: inherit;
  }
```

### What did you expect to happen?

`inherit` is either ignored by default or there is a configuration option to ignore it.

### What actually happened?

```console
error  Use only allowed relative units for 'font-size' - rem    css/relative-font-unit
```

### Link to Minimal Reproducible Example

https://github.com/eslint/css

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

It might make sense to also be able to ignore the other global values:

```
/* Global values */
font-size: inherit;
font-size: initial;
font-size: revert;
font-size: revert-layer;
font-size: unset;
```
https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#syntax

**Comment by nzakas** [comment]
@xbinaryx I think that's a good idea. :+1:

While this isn't a bug, I do think adding an option makes sense.

**Comment by nzakas** [comment]
@sdavids it's unclear from your description how you'd like the rule to change, because it appears that you've listed `inherit` and other values as things the rule *shouldn't* care about?

It would be helpful if you gave a concrete proposal about the change you'd like to see in this rule.

**Comment by nzakas** [comment]
@sdavids I don't know how I'm supposed to interpret this code example.

**Comment by sdavids** [comment]
Given the config 

```javascript
      "css/relative-font-units": [
        "error",
        {
          allowUnits: ["rem"],
        },
      ],
```

and the CSS file


```css
  h1 {
    font-size: inherit; /* ERROR */
  }

  h2 {
    font-size: xxx-large; /* ERROR */
  }

  h3 {
    font-size: 3rem; /* OK */
  }

  h4 {
    font-size: 3em; /* ERROR */
  }
```

ESLint/CSS will report errors for the lines marked with `/* ERROR */`.

Same for the other example …

**Comment by sdavids** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/length
https://developer.mozilla.org/en-US/docs/Web/CSS/percentage

All others should be ignored.

**Comment by nzakas** [comment]
@Tanujkanti4441 I don't think it makes sense to group "all others" as that encapsulates a lot of different value types.

Here's what I think we should do:

1. `<percentage>` is always allowed.
2. `smaller` and `larger` are always allowed.
3. `<length>` is allowed so long as the units are relative. Units other than those allowed are errors.
4. Global values (`inherit`, `unset`, etc.) are always allowed.
5. `<absolute-size>` is always an error.

So, no new options, just changing the defaults. We can always add options later if necessary.
